Sources/StreamChat/WebSocketClient/Events/MessageEvents.swift

```diff
@@ -4,78 +4,168 @@
 
 import Foundation
 
-public struct MessageNewEvent: MessageSpecificEvent {
-    public let userId: UserId
-    public let cid: ChannelId
-    public let messageId: MessageId
+public struct MessageNewEvent: Event {
```

Comment:
For some reason, there's not such event on `Android`, they have only `NotificationMessageNewEvent`. Should we do the same?